### DISCUSSION TITLE
Noob

### DISCUSSION THREAD

**Discussion by baham0617** [jaykali/maskphish#64] [Q&A] [answered]
I'm very new to this whole process. To my own surprise I have installed as well as utilized maskphish, Advphisher and Nexphisher in just 6 hours. Now I'm at a loss. I've generated the URL via maskphish but I don't know how to send it to the victim anonymously and how do I know when they receive it or hit on the link?

**Comment by jaykali** [comment, marked as answer]
There are lots of options available. You can use anonymous mails. Telegram also can hide your original identity. Also you can easily get virtual mobile numbers. Paid bulk SMS services also can be an option.

**Reply by baham0617** [reply]
Thank you very much. Do i just go back into msskphish to see if the link was accessed?

**Reply by jaykali** [reply]
> Thank you very much. Do i just go back into msskphish to see if the link was accessed?

Nope. MaskPhish just modify the URL and give it a trust-able look. It doesn't track anything.

**Reply by baham0617** [reply]
Ok thanks
